cmake/compilers/Clang.cmake

```diff
@@ -50,8 +57,7 @@ if (CMAKE_SYSTEM_PROCESSOR MATCHES "(AMD64|amd64|i.86|x86)")
 endif()
 
 # Clang flags to prevent compiler from optimizing out security checks
-set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -Wformat -Wformat-security -Werror=format-security
-	                                                  -fstack-protector-strong -fPIC)
+set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -Wformat -Wformat-security -Werror=format-security -fPIC $<$<NOT:$<IF:${EMSCRIPTEN},1,0>>:-fstack-protector-strong>)
```

Comment:
```suggestion
set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -Wformat -Wformat-security -Werror=format-security -fPIC $<$<NOT:EMSCRIPTEN>:-fstack-protector-strong>)
```

Reply:
Reverted it back. Wasn't working.

Reply:
`IF` approach still feels as an overkill though. Can you try this option?
```
$<$<NOT:$<BOOL:${EMSCRIPTEN}>>:-fstack-protector-strong>
```